cloud/api/protobuf/kurtosis_backend_server_api.proto

```diff
@@ -222,14 +223,17 @@ message User {
   string Updated = 4;
 }
 
-message GetUnlockedPortsRequest {
+message GetPortsMetadataRequest {
```

Comment:
Enriching existing requests and reusing the ports table felt the easiest; 

https://github.com/kurtosis-tech/kurtosis-cloud-backend/pull/154 implementation here